### PR TITLE
Updated methods and examples for business accounts

### PR DESCRIPTION
**Pull request by jotathebest**
implemented new methods for subscribe and publish properly to busines…s accounts and show debug messages. Updated new and old examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ reconnect();
 > For trying to make a reconnection every 5 seconds if the connection is lost.
 
 ```
+ubidotsSetBusiness(bool business);
```
Comment (woakas) [inline README.md]: This method should not go, with the setBroker or or setHost method should be sufficient

File: UbidotsESPMQTT.cpp [hunk 2]
```diff
@@ -148,3 +153,19 @@ bool Ubidots::wifiConnection(char* ssid, char* pass) {
     Serial.println(F("IP address: "));
     Serial.println(WiFi.localIP());
 }
+
```
Comment (woakas) [inline UbidotsESPMQTT.cpp]: This methos is not necessary

File: UbidotsESPMQTT.h [hunk 3]
```diff
@@ -57,19 +57,23 @@ class Ubidots {
     char* _clientName;
     char* _token;
     uint8_t currentValue;
+    char* _server;
+    bool _debug = true;
```
Comment (woakas) [inline UbidotsESPMQTT.h]: by default must be false

**Comment by jotathebest** [comment]
Fixed. Updated examples and keywords.

**Pull request merged**